performance/us/kbase/workspace/performance/refsearch/GetReferencedObjectWithBFS.java

```diff
@@ -24,7 +24,6 @@
 import us.kbase.typedobj.idref.IdReferenceHandlerSetFactory;
```

Comment:
Basically all the test code here really needs some extreme TLC; it's been neglected for years. I did a bit of that here and there in a halfhearted, inconsistent manner when I saw it, and it's better than before, but I quickly realized that trying to really clean things up would explode this (already too big) PR so I decided to more or less leave things alone unless I was touching it at the time.